velox/connectors/Connector.h

```diff
@@ -226,6 +226,17 @@ class DataSource {
   virtual int64_t estimatedRowSize() {
     return kUnknownRowSize;
   }
+
+  virtual std::shared_ptr<common::Filter> getFilter(
```

Comment:
This PR changes the Connector API. Please, update PR description to describe the changes and add comments for the newly introduced APIs.

Reply:
Updated, thanks.